docs/DEPLOYMENT.md

```diff
@@ -144,7 +180,7 @@ After the deployment is done successfully, we need to expose the URL to access t
 - If deploying on OpenShift:-
 
   ```bash
-  oc apply -f config/openshift/
+  kubcetl apply -f -f 04-openshift/40-api-route.yaml
```

Comment:
```suggestion
  kubcetl apply -f  04-openshift/40-api-route.yaml
```

Reply:
I guess we can keep `oc` here as it is specific to openshift. WDYT?
```suggestion
  oc apply -f 04-openshift/40-api-route.yaml
```

Reply:
btw the spelling of `kubectl` is wrong over there :stuck_out_tongue:

Reply:
kubectl is fine I guess.. it can apply route 😄